### ISSUE TITLE
Development of the site

### ISSUE THREAD

**Issue by atharva3010** [whynotearth/shinta-mani-wild#12]
The figma file for the UX is ready: https://www.figma.com/file/SiFZE7hhRKx2fWmrfZ3uy2RO/Shinta-Mani-Wild?node-id=7%3A25  
  
We can begin the development of the app by first creating a basic layout of the page and then following the following order of development:  

- Card component creation
- Button component creation
- header component
- banner image component
- toolbar and footer
- login/sign up components
- using our swagger to axios codegen
- setting up store and filling up all the data
- payment integration

**Comment by paulchrisluke** [comment]
I've broken this out into a project file here, we may want to make each card an issue? https://github.com/whynotearth/shinta-mani-wild/projects

**Comment by StrangeWill** [comment]
> @mort3za @StrangeWill this project can be a good start to shift to typescript with Vue?

Yep, honestly any of our sites can be retrofitted for TypeScript and have them slowly migrated too.

> I've broken this out into a project file here, we may want to make each card an issue? 

Man Github projects is so weird in my mind -- usually I'd start with issues first, the cards are almost weird as a thing that doesn't really exist anywhere (that's just me from the software development lifecycle side of things though)

**Comment by mort3za** [comment]
@StrangeWill It was weird to me too but I think that separate cards are a good idea because every issue can be split into several cards, and also we can create cards from issues too.
